torchscript_export.py

```diff
@@ -0,0 +1,109 @@
+import os
+
+import sys
+
+sys.path.append("core")
+
+import argparse
+import torch
+from monster import Monster
+from utils.utils import InputPadder
+
+
+class ScriptableMonSter(torch.nn.Module):
+    def __init__(self, monster: torch.nn.Module):
+        super().__init__()
+        self.monster = monster
+
+    def forward(
+        self, left_image: torch.Tensor, right_image: torch.Tensor
+    ) -> torch.Tensor:
```

Comment:
Let's make sure we follow the style guide, please.
```suggestion
    def forward(
        self,
        left_image: torch.Tensor,
        right_image: torch.Tensor,
    ) -> torch.Tensor:
```